internal/providers/terraform/google/logging_billing_account_bucket_config.go

```diff
@@ -0,0 +1,46 @@
+package google
+
+import (
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/shopspring/decimal"
+)
+
+func GetLoggingBillingAccountBucketConfigRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "google_logging_billing_account_bucket_config",
+		RFunc: NewLoggingBillingAccountBucket,
+	}
+}
+
+func NewLoggingBillingAccountBucket(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
```

Comment:
Can we please put this logic in a function so all of the resources that need this cost component can just call that same function? [DRYs](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) the code a bit.